util.py

```diff
@@ -0,0 +1,80 @@
+# Enter your source directory between the quotes here
+SOURCE_DIRECTORY = ''
```

Comment:
Im not strong on best practices for keeping user config values separate from other utility functions. Could keep as setup.py and somehow reference path from util.py, but could be awkward.

Reply:
The only reason for the two to be consolidated into util is because TestWrapper class requires the full path to bitcoin directory. Setup.py adds it to PATH. I guess I could get it from the PATH var somehow in util.py? But that’s a little awkward.

Reply:
Yeah, the only reason they are separate is because I wanted to avoid referenceh path from util.py, after setup.py sets source directory.